src/components/VerifiedFieldComponents.tsx

```diff
@@ -0,0 +1,174 @@
+import CheckIcon from "@mui/icons-material/Check";
+import {
+  Box,
+  Checkbox,
+  Divider,
+  IconButton,
+  InputBase,
+  Tooltip,
+  Typography,
+} from "@mui/material";
+import { ReactNode, useState } from "react";
+import { Control, Controller, useFormContext, useWatch } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+
+function VerifiedFieldWrapper({
+  label,
+  path,
+  className = "",
+  renderField,
+}: {
+  label: string;
+  path: string;
+  className?: string;
+  renderField: (props: {
+    setIsFocused: (value: boolean) => void;
+    control: Control;
+    valuePath: string;
+    verified: boolean;
+  }) => ReactNode;
+}) {
+  const { t } = useTranslation("labelDataValidationPage");
```

Comment:
Use arrow function syntax

Define an interface instead of having it inline for readability / consistency with other functions